### PR TITLE
Navbar Home Page Title

### PR DESCRIPTION
**Pull request by catnoel**
i change the navbar title from Home to Home is where the heart is.

### CODE REVIEW

**Comment by PhillipDaum** [comment]
I know home is where the heart is, but I think that is really long for a button title.

**Review by aj1441 — APPROVED** [review]
ok this is good

**Review by PhillipDaum — COMMENTED** [review]
yes

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ function Navbar() {
             <nav className='changing-nav'>
                 <ul>
                     <li>
-                        <Link to="/">Home</Link>
+                        <Link to="/">Home is where the heart.</Link>
```
Comment (ArcieeX) [inline src/components/Navbar.jsx:28]: i think we agrred to home is far away

Comment (catnoel) [reply src/components/Navbar.jsx:28]: You're correct, i will change that now!

**Review by aj1441 — APPROVED** [review]

**Comment by PhillipDaum** [comment]
looks good!

**Review by ArcieeX — APPROVED** [review]
looks good

**Pull request merged**